runtime/criusupport/criusupport.cpp

```diff
@@ -938,6 +938,8 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		}
 
 		if (NULL != vm->checkpointState.restoreArgsList) {
+			/* mark -Xoptionsfile= as consumed */
+			FIND_AND_CONSUME_ARG(vm->checkpointState.restoreArgsList, STARTSWITH_MATCH, VMOPT_XOPTIONSFILE_EQUALS, NULL);
```

Comment:
is this the only necesarry change here?

Reply:
There were two other issues https://github.com/eclipse-openj9/openj9/pull/16873#discussion_r1131687902
Afterwards, this change is required to fix following `JVMRestoreException`.
```
JVMJ9VM007E Command-line option unrecognised: -Xoptionsfile=/root/tests/jdk11-criusysprops-0309/aqa-tests/TKG/options
Exception in thread "main" org.eclipse.openj9.criu.JVMRestoreException: The JVM could not enable all the restore options specified
	at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVMImpl(Native Method)
	at openj9.criu/org.eclipse.openj9.criu.CRIUSupport.checkpointJVM(CRIUSupport.java:635)
	at org.openj9.criu.CRIUTestUtils.checkPointJVM(CRIUTestUtils.java:77)
	at org.openj9.criu.OptionsFileTest.propertiesTest1(OptionsFileTest.java:65)
	at org.openj9.criu.OptionsFileTest.main(OptionsFileTest.java:38)
```